src/js/repeat.js

```diff
@@ -133,40 +133,23 @@ export default {
             return 0;
         }
 
-        let checkEl = el.parentElement.closest( '.or-repeat' );
```

Comment:
There's currently no test coverage for a case where the `closest` call would return something other than the element itself. As far as I can tell, the function should always be given a repeat or repeat info so that would never happen. I could be missing something, though, and this seems it could be risky.